jsonbox-api/tests/jsons/test_api.py

```diff
@@ -2,14 +2,14 @@
 from rest_framework import status
 from rest_framework.reverse import reverse
 
-from apps.jsons.models import Json
+from .factories import JsonFactory
 
 pytestmark = pytest.mark.django_db
 
 
 @pytest.fixture
 def sample_json(box):
-    js = Json(box=box, data={"key": "value", "lol": {"name": "hue", "age": 1}})
+    js = JsonFactory(box=box, data={"key": "value", "lol": {"name": "hue", "age": 1}})
```

Comment:
```suggestion
    return JsonFactory.create(box=box, data={"key": "value", "lol": {"name": "hue", "age": 1}})
```

I don't remember whether the right method is `create` or `build`, but on of those already saves the model on the db. By using it the two lines below aren't necessary anymore.

Reply:
@lamenezes `create` save the instance, `build` just, well, build :P
Also, just calling the factory should save too. [1] 

[1] https://factoryboy.readthedocs.io/en/latest/introduction.html#strategies